oci.go

```diff
@@ -245,8 +260,15 @@ func processCgroupsPathForResource(ociSpec oci.CompatOCISpec, resource string) (
 		}
 	}
 
-	if cgroupMountFound == false {
-		return "", fmt.Errorf("cgroupsPath is absolute, cgroup mount MUST exist")
+	if !cgroupMountFound {
+		if isPod {
+			return "", fmt.Errorf("cgroupsPath %q is absolute, cgroup mount MUST exist",
```

Comment:
I appreciate this isn't strictly part of this PR, but I find this error rather confusing - the implication is that the path doesn't exist, but what about the absolute bit? Is that a problem too? Could the error become something like this?:

```
"absolute cgroupsPath %q does not exist"
```